tls/s2n_signature_algorithms.h

```diff
@@ -29,12 +29,12 @@ struct s2n_sig_scheme_list {
     uint8_t len;
 };
 
-extern int s2n_get_signature_scheme_pref_list(struct s2n_connection *conn, const struct s2n_signature_scheme* const** pref_list_out,
-                                              size_t *list_len_out);
-
-extern int s2n_choose_sig_scheme_from_peer_preference_list(struct s2n_connection *conn, struct s2n_sig_scheme_list *sig_hash_algs,
+int s2n_choose_sig_scheme_from_peer_preference_list(struct s2n_connection *conn, struct s2n_sig_scheme_list *sig_hash_algs,
                                                             struct s2n_signature_scheme *sig_scheme_out);
-extern int s2n_get_and_validate_negotiated_signature_scheme(struct s2n_connection *conn, struct s2n_stuffer *in,
+int s2n_get_and_validate_negotiated_signature_scheme(struct s2n_connection *conn, struct s2n_stuffer *in,
```

Comment:
AFAIK, extern has no real effect here. I've been using it to keep consistent with other header files. Is there a specific reason to remove it, or is this just preference?

Reply:
It's redundant. All functions definitions are extern by default. I don't know when we started putting extern in front of functions, but it just makes things more verbose / noisy and we should really stop.